.github/workflows/build.yml

```
@@ -186,20 +186,64 @@ jobs:
       - name: Code Quality Checks
         run: ./gradlew qualityCheck :examples:qualityCheck --continue --rerun-tasks
 
-      - name: Run Unit and Integration Tests
-        env:
-          OPERATOR_KEY: "302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137"
-          OPERATOR_ID: "0.0.2"
-          HEDERA_NETWORK: "localhost"
+      - name: Run Unit and Integration Tests (excluding NodeUpdateTransactionIntegrationTest)
         run: |
-          ./gradlew -POPERATOR_ID=$OPERATOR_ID -POPERATOR_KEY=$OPERATOR_KEY -PHEDERA_NETWORK=$HEDERA_NETWORK :aggregation:testCodeCoverageReport
+          ./gradlew \
+            -POPERATOR_ID=$OPERATOR_ID \
+            -POPERATOR_KEY=$OPERATOR_KEY \
+            -PHEDERA_NETWORK=$HEDERA_NETWORK \
+            :aggregation:testCodeCoverageReport \
+            --exclude-test "*NodeUpdateTransactionIntegrationTest"
 
       - name: Upload coverage to Codecov
         if: ${{ github.event_name == 'push' || (github.event.pull_request.head.repo.full_name == github.repository && github.actor != 'dependabot[bot]') }}
         uses: codecov/codecov-action@5c47607acb93fed5485fdbf7232e8a31425f672a # v5.0.2
         with:
           files: gradle/aggregation/build/reports/jacoco/testCodeCoverageReport/testCodeCoverageReport.xml
 
+  test-node-update-account-id:
```

```suggestion
  test-dab:
```
dab stands for dynamic addressbook